config/initializers/ab_tests.rb

```diff
@@ -46,7 +46,6 @@ def self.in_person_cta_variant_testing_buckets
 
   IN_PERSON_CTA = AbTestBucket.new(
     experiment_name: 'In-Person Proofing CTA',
-    buckets: in_person_cta_variant_testing_buckets,
     default_bucket: 'in_person_variant_a',
   )
```

Comment:
we're since there's only one option, should we remove this entire AB test?

Reply:
yes i think you're right